app/exerlog/lib/src/widgets/gradient_button.dart

```diff
@@ -43,7 +43,7 @@ class _RaisedGradientButtonState extends State<RaisedGradientButton> {
           borderRadius: BorderRadius.circular(widget.radius),
         ),
         child: TextButton(
-          onPressed: _onPressed(),
+          onPressed: () => _onPressed(),

```

Comment:
Sorry last one haha, but I believe you can just do this now:

`onPressed: _onPressed,`